### PR TITLE
compute: implement machineType

### PR DESCRIPTION
**Pull request by ashleyschuett**
For #1073

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Thank you so much!

File: system-test/compute.js [hunk 1]
```diff
@@ -1061,6 +1061,31 @@ describe('Compute', function() {
...
+        assert.ifError(err);
+        assert(machineTypes.length > 0);
+        done();
+      });
+    });
+
+    it('should get a list of machineTypes in stream mode', function(done) {
+      var resultsMatched = 0;
+
+      compute.getZones()
```
Comment (Gormartsen) [inline system-test/compute.js]: Sounds like typo. Probably should be `compute.getMachineTypes()`

**Comment by stephenplusplus** [comment]
PR'd the PR for some small stuff: https://github.com/containership/gcloud-node/pull/3

**Comment by stephenplusplus** [comment]
Yay, Travis is happy! This PR is good to go, however we're in a small code freeze as we prepare for the modularization shift introduced in #1431. After that lands, I'll do the converting to the new type and send yet another PR to your fork.

**Comment by stephenplusplus** [comment]
Merged via https://github.com/GoogleCloudPlatform/gcloud-node/commit/0f468791521c677fdcfb45fd601ab3049e81e9e1 -- thanks again!

**Pull request closed without merging**